src/service/viewport-impl.js

```diff
@@ -140,6 +143,16 @@ export class Viewport {
 
     this.binding_.onScroll(this.scroll_.bind(this));
     this.binding_.onResize(this.resize_.bind(this));
+
+    this.onScroll(() => {
+      if (!this.duringAnimationFrame_) {
+        this.duringAnimationFrame_ = true;
+        this.vsync_.measure(() => {
+          this.duringAnimationFrame_ = false;
+          this.viewer_.postScroll(this.binding_.getScrollTop());
```

Comment:
I already throttled this call per animation frame. Isn't that enough? The `scroll_()` function in `viewport-impl` called `this.binding_.getScrollTop()` every time it scrolls. See [here](https://github.com/ampproject/amphtml/pull/3421/files/cae69cc333c61757ad96c44592a86ee387c1f98c#diff-6eedd5ca45df1b5aee7d616b017d360aL493). Isn't that more expensive?